l1-contracts/deploy-scripts/RegisterCTM.s.sol

```diff
@@ -169,19 +195,23 @@ contract RegisterCTM is Script, DeployUtils {
             salt: bytes32(0)
         });
 
-        governance.scheduleTransparent(operation, 0);
-        // We assume that the total value is 0
-        governance.execute{value: 0}(operation);
+        if (shouldSend) {
+            governance.scheduleTransparent(operation, 0);
+            // We assume that the total value is 0
+            governance.execute{value: 0}(operation);
 
-        console.log("CTM DT whitelisted");
-        vm.stopBroadcast();
+            console.log("CTM DT whitelisted");
+            vm.stopBroadcast();
 
-        bytes32 assetId = bridgehub.ctmAssetIdFromAddress(addresses.stateTransition.chainTypeManagerProxy);
-        console.log(
-            "CTM in router 1",
-            sharedBridge.assetHandlerAddress(assetId),
-            bridgehub.ctmAssetIdToAddress(assetId)
-        );
+            bytes32 assetId = bridgehub.ctmAssetIdFromAddress(addresses.stateTransition.chainTypeManagerProxy);
+            console.log(
+                "CTM in router 1",
+                sharedBridge.assetHandlerAddress(assetId),
+                bridgehub.ctmAssetIdToAddress(assetId)
+            );
+        } else {
+            saveOutput(Output({admin: address(governance), encodedData: abi.encode(calls)}), outputPath);
```

Comment:
why this logic is correct, it would be better if we did not call decentralized governance `admin` as it may confuse people. 

especially since you anyway reimplement `saveOutput` and `Output` struct, so not code deduplication was won here